editoast/src/views/v2/train_schedule.rs

```diff
@@ -676,6 +709,35 @@ mod tests {
         assert!(response.status().is_client_error());
     }
 
+    #[rstest]
+    async fn get_batch_trainschedule(
+        #[future] train_schedule_v2: TrainScheduleV2FixtureSet,
+        db_pool: Data<DbConnectionPool>,
+    ) {
+        let service = create_test_service().await;
+        let fixture = train_schedule_v2.await;
+        let url = format!("/v2/train_schedule/?ids[]={}", fixture.train_schedule.id);
```

Comment:
Minor (if it's worth the time): how about testing multiple values + testing error case ?

Reply:
Done (both)